criterion-measurement/src/Criterion/Measurement.hs

```diff
@@ -32,9 +32,10 @@ module Criterion.Measurement
     , measured
     , applyGCStatistics
     , threshold
-    ) where
+    )
+    where
```

Comment:
In general, I'd prefer not to clutter up the diff with minor aesthetic changes like this unless there's a good reason for them (which there doesn't seem to be here).